pipeline/direct/direct.py

```diff
@@ -352,22 +354,35 @@ def get_hazard(haz_type, country_iso3alpha, scenario, ref_year):
         )
         return haz
 
-    elif haz_type == "relative_crop_yield":
+    elif haz_type.startswith("relative_crop_yield"):
+        _, crop_type = agriculture.split_agriculture_hazard(haz_type)
         # TODO currently always returns the same hazard
         if scenario == 'None' and ref_year == "historical":
-            return agriculture.get_hazard(
-                country=country_iso3alpha,
-                year_range="1971_2001",
-                scenario="historical"
+            #For soy, there is another historical period available (due to availability)
+            if crop_type == 'soy':
+                return agriculture.get_hazard(
+                    country=country_iso3alpha,
+                    year_range="1980_2012",
+                    scenario="historical",
+                    crop_type=crop_type
+                )
+            #For the other crop types we use this historical period (due to availability)
+            else:
+                return agriculture.get_hazard(
+                    country=country_iso3alpha,
+                    year_range="1971_2001",
+                    scenario="historical",
+                    crop_type=crop_type
             )
         else:
             return agriculture.get_hazard(
                 country=country_iso3alpha,
                 year_range="2006_2099",
```

Comment:
The hazard we get here from the API is from long-duration ISIMIP simulations for the period 2006-2099. Should we think about subsetting it to be more representative of 2060?

The usual thing to do would be to extract the 30 years around 2060. This would still give us about 400 years of simulated crop loss which is good enough for our work. (We could also grab a 40-year interval for a larger event set).

The other option would be grabbing the period 2050-2099 and then scaling the impacts 'back' to 2060. The reason I suggest this is that ISIMIP's crop yield climate change signal is pretty weak and pretty noisy. By taking years further in the future we have a better chance of capturing the signal. This might be a bit more work, so we could also leave this as a nice-to-have if there's time.

Reply:
Hey Chris, 

I did not know that we could extract years out of this range. I thought it was this period for which the value was inserted and I was already wondering why they chose such a big range. If we can extract individual years or the range you suggested, I think it would be a nice thing to do. Do you by chance know how we can split this hazard easily?

--> To clarify if this is for the MVP

Reply:
I think CLIMADA has a method for this. Once you've loaded the hazard `haz` from the data API you can subset by date with
```
haz = haz.select(date=('1940-01-01', '1959-12-31')
```
I think it's worth doing for the MVP (it'll also speed up the impact calculation just a little 😁 )

Reply:
Ha, I wasn't aware of this either, thanks @ChrisFairless